packages/browser/src/sdk.ts

```diff
@@ -37,6 +38,9 @@ export function getDefaultIntegrations(_options: Options): Integration[] {
     dedupeIntegration(),
     httpContextIntegration(),
     browserSessionIntegration(),
+    /* development-only */
+    spotlightBrowserIntegration(),
+    /* end-development-only */
```

Comment:
l: let's try to point it out a bit more clearly that this comment block must not be removed/changed. Feel free to go with something else but I was thinking of something along these lines:

```suggestion
    /* rollup-include-development-only */
    spotlightBrowserIntegration(),
    /* end-rollup-include-development-only */
```